src/backend/tests/work-packages.test.ts

```diff
@@ -91,4 +112,26 @@ describe('Work Packages', () => {
     expect(res.statusCode).toBe(400);
     expect(res.body.message).toBe('One of the dependencies was not found.');
   });
+
+  test('getChangeRequestReviewState returns null when changeRequest is not found', async () => {
+    jest.spyOn(prisma.user, 'findUnique').mockResolvedValue(batman);
+    jest.spyOn(prisma.change_Request, 'findUnique').mockResolvedValue(null);
+    const spy = jest.spyOn(Utils, 'getChangeRequestReviewState');
+
+    const res = await request(app).post('/create').send(createWorkPackagePayload);
+
+    expect(spy).toHaveBeenCalledWith(createWorkPackagePayload.crId);
+    expect(spy).toHaveReturnedWith(null);
+  });
+
+  test('getChangeRequestReviewState returns false when changeRequest has not been reviewed', async () => {
+    jest.spyOn(prisma.user, 'findUnique').mockResolvedValue(batman);
+    jest.spyOn(prisma.change_Request, 'findUnique').mockResolvedValue(unreviewedCr);
+    const spy = jest.spyOn(Utils, 'getChangeRequestReviewState');
+
+    const res = await request(app).post('/create').send(createWorkPackagePayload);
+
+    expect(spy).toHaveBeenCalledWith(createWorkPackagePayload.crId);
+    expect(spy).toHaveReturnedWith(false);
```

Comment:
idk why this isn't working, the actual values return as expected (from console.log testing), but jest is saying an empty object was returned.